source/Superluminal-RESTfulAPI/ExpiringCacheKeyBuilder.class.st

```diff
@@ -48,7 +48,7 @@ ExpiringCacheKeyBuilder >> build [
 { #category : 'configuring' }
 ExpiringCacheKeyBuilder >> entity: entity [
 
-	
+  entity contents ifNotEmpty: [ :contents | parameters at: #entity put: contents asString ]
```

Comment:
I'm not sure this will work for every kind of entity. Take into account that entities can be binary, gzipped, not just text. At least add some tests for the binary cases to see if this doesn't break, or if is not our intention to support non-textual entities raise a proper error to the user. You can ask if the entity `hasContentType` and in that case if the `contentType isBinary` to make the distinction if needed.